### PR TITLE
Fix bug retrieving from LDAP which not uses decorete_core_limit and then no scim info in in hints

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,6 +1,8 @@
 CHANGES
 =======
 
+Fix bug retrieving from LDAP which not uses decorete_core_limit and then no scim info in in hints
```
Comment (fgalan) [inline ChangeLog:4]: No issue number associated?

Comment (AlvaroVega) [reply ChangeLog:4]: No issue

Comment (fgalan) [reply ChangeLog:4]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**